substrate/utils/frame/remote-externalities/src/lib.rs

```diff
@@ -1255,7 +1265,7 @@ mod tests {
 
 		Builder::<Block>::new()
 			.mode(Mode::Offline(OfflineConfig {
-				state_snapshot: SnapshotConfig::new("test_data/proxy_test"),
+				state_snapshot: SnapshotConfig::new("test_data/test.snap"),
```

Comment:
<img width="148" alt="Screenshot 2024-05-06 at 17 00 21" src="https://github.com/paritytech/polkadot-sdk/assets/10380170/58c2b48a-64f3-42af-a06a-6ca35aff5229">

Dunno if something like this could be worth it: https://docs.rs/include-bytes-zstd/latest/include_bytes_zstd/

Reply:
Git is pretty efficient, so I'd say not in this case :)